Formula/azure-cli-2.rb

```diff
@@ -0,0 +1,63 @@
+require "tmpdir"
+
+class AzureCli2 < Formula
+  include Language::Python::Virtualenv
+
+  desc "Azure CLI 2.0"
+  homepage "https://github.com/Azure/azure-cli"
+  url "https://azurecliprod.blob.core.windows.net/releases/azure-cli_packaged_0.1.6.tar.gz"
+  sha256 "bb4bd4694e044bf416941e3afdc505bde7d049bac2e66c8b3fe44c9cb3b57ff8"
+  depends_on "python"
+
+  # Apply the 'az component' patch
+  patch do
+    url "https://azurecliprod.blob.core.windows.net/patches/patch_2_component_custom.diff"
+    sha256 "d61ef29ace9bbdfef9a25dfbb1f475225bbca174263c8f863ee70f87d0a78bbe"
+  end
+
+  # Apply pip module search patch
```

Comment:
Has this patch been merged/submitted upstream? Regardless, can you link to the upstream merge/submission as a Ruby comment here, thanks!